datacontract/utils.py

```diff
@@ -0,0 +1,86 @@
+import json
+import logging
+import typing
+
+import yaml
+from datacontract.model.data_contract_specification import DataContractSpecification, Server
+
+
+def _determine_sql_server_type(data_contract: DataContractSpecification, sql_server_type: str, server: str = None):
```

Comment:
No "utils.py" files please (especially not on top-level). After one day nobody knows what's in there.